appveyor.yml

```diff
@@ -53,6 +53,7 @@ on_success:
 on_failure:
   - echo "*** FAILURE ***"
 on_finish:
+  - ps: (new-object net.webclient).UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\test_results.xml))
```

Comment:
Hm, not sure -- I copied this from:
https://github.com/mgerhardy/caveexpress/blob/master/appveyor.yml#L53

Looks like net.webclient may work (assuming case insensitive):
https://www.appveyor.com/docs/how-to/download-file/#webclient-class